addons/delivery/models/stock_picking.py

```diff
@@ -200,7 +200,7 @@ def check_packages_are_identical(self):
         self.ensure_one()
         if self.package_ids:
             packages = [p.packaging_id for p in self.package_ids]
-            if len(set(packages)) != 1:
+            if len(set([package.shipper_package_code for package in packages])) != 1:
```

Comment:
can be also expressed `packages.mapped('shipper_package_code')`

Reply:
"packages.mapped('shipper_package_code')"
Here packages is a 'list' object.We can't use 'mapped'.